packages/blade/src/components/Dropdown/useDropdown.ts

```diff
@@ -79,12 +86,12 @@ type DropdownContextType = {
    */
   hasFooterAction: boolean;
   setHasFooterAction: (value: boolean) => void;
+
   /**
-   * We need to know the label's position because when it is on left, the overlay takes the width of input.
-   * Rest of the times, we can set 100% width
+   * Apart from dropdownTriggerer prop, we also set this boolean because in BottomSheet, the initial trigger can be Select but also have autocomplete inside of it
    */
-  hasLabelOnLeft: boolean;
-  setHasLabelOnLeft: (value: boolean) => void;
+  hasAutoCompleteInBottomSheetHeader: boolean;
```

Comment:
Isn't this changing the existing behaviour for `hasLabelOnLeft`?

Reply:
ops sorry for confusion. I removed `hasLabelOnLeft` because that type wasn't used anymore. floating-ui migration of DropdownOverlay removed its purpose but it existed in DropdownContext.

Reply:
Basically removing `hasLabelOnLeft` is unrelated to this PR